packages/assert/src/assert.js

```diff
@@ -162,7 +162,7 @@ function details(template, ...args) {
         parts.push('\nSee console for error data.');
       }
       const err = new Error(parts.join(''));
-      console.error('LOGGED ERROR:', ...interleaved, err);
+      // console.error('LOGGED ERROR:', ...interleaved, err);
```

Comment:
You could do something like the following. It can be disabled by default, as long as there's some way to enable it.

```js
if (someAppropriateOptionSetSomehow) {
  console.error('LOGGED ERROR:', ...interleaved, err);
}
```

Reply:
https://github.com/Agoric/SES-shim/issues/440

Reply:
Thanks, this all makes sense.